x/ccv/consumer/keeper/throttle_retry.go

```diff
@@ -0,0 +1,109 @@
+package keeper
+
+import (
+	"fmt"
+	"time"
+
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
```

Comment:
nit: Not sure what's the naming convention here, but in `relay_test.go` we have:
```
	sdk "github.com/cosmos/cosmos-sdk/types"
```
Should we have `sdk` here as the name to be consistent?

Reply:
```
	sdk "github.com/cosmos/cosmos-sdk/types"
```

is probably used most often. I personally try to use the `sdktypes` alias as it's more descriptive of what path we're actually referencing